index.html

```diff
@@ -32,22 +32,22 @@
         <div class="container" id="home">
             <div class="grid-container">
                 <figure class="item1">
-                    <img src="assets/images/burger.jpg" class="imageps hvr-grow" alt="Image 1">
+                    <img src="assets/images/burger.jpg" class="imageps hvr-grow" alt="Image 1" />
```

Comment:
what happens if u remove `imageps` ? does it keep working?

Reply:
It is important because it has a css property associated.

`.imageps {
    width: 100%;
    height: 100%;
    object-fit: cover;
    display: block;
}`